### PR TITLE
5702. Styling block breaking

### PR DESCRIPTION
**Pull request by bobik-dev-branck**
LInk to redmine - https://redmine.devbranch.work/issues/5702

### CODE REVIEW

File: web/themes/custom/newsweek/scss/_mixins.scss [hunk 1]
```diff
@@ -12,3 +12,49 @@
...
+      display: inline;
+      background-color: $main-color-red;
+      color: #fff !important;
+      font-family: Work Sans;
+      font-size: 15px;
+      padding: 2px;
+      border-radius: 2px;
+      text-transform: uppercase;
+      line-height: 0;
+      > a :hover {
```
Comment (Y0r) [inline web/themes/custom/newsweek/scss/_mixins.scss:33]: &

File: web/themes/custom/newsweek/scss/_mixins.scss [hunk 2]
```diff
@@ -12,3 +12,49 @@
...
+    }
+  }
+}
+
+@mixin view-mode_meta-info {
+  .field--name-title {
+    font-family: "PT Serif";
+    font-size: 14px;
+    font-weight: 600;
+    }
```
Comment (Y0r) [inline web/themes/custom/newsweek/scss/_mixins.scss:45]: space

File: web/themes/custom/newsweek/scss/content/_breaking.scss [hunk 3]
```diff
@@ -0,0 +1,137 @@
+.breaking {
+  background-color: $main-color-grey;
+  position: relative;
+  margin: 0 -10px;
+  .contextual-region {
+    padding: 0 10px;
+  }
+
+  //Styles for Breaking block title.
```
Comment (Y0r) [inline web/themes/custom/newsweek/scss/content/_breaking.scss:9]: CS

File: web/themes/custom/newsweek/scss/content/_breaking.scss [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    height: 46px;
+    text-align: center;
+    line-height: 44px;
+    position: absolute;
+    top: -15px;
+    left: 40px;
+    z-index: 5;
+  }
+
+  //Styles for Breaking block.
```
Comment (Y0r) [inline web/themes/custom/newsweek/scss/content/_breaking.scss:29]: CS

File: web/themes/custom/newsweek/scss/content/_breaking.scss [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+            &:hover {
+              color: #d3d2d2;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  //Styles for Breaking block attachment.
```
Comment (Y0r) [inline web/themes/custom/newsweek/scss/content/_breaking.scss:95]: CS

File: web/themes/custom/newsweek/templates/content/block--views-block--breaking-block-1.html.twig [hunk 6]
```diff
@@ -0,0 +1,5 @@
+{% import '@newsweek/content/node--newsweek-article--teaser.html.twig' as content %}
+<div class="breaking__title">
+  Breaking
```
Comment (Y0r) [inline web/themes/custom/newsweek/templates/content/block--views-block--breaking-block-1.html.twig:3]: {{ 'Breaking'|t  }}

File: web/themes/custom/newsweek/templates/content/block--views-block--fresh-stories-block-1.html.twig [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+  ]
+%}
+<div{{ attributes.addClass(classes) }}>
+  {{ title_prefix }}
+  {% if label %}
+    <h2{{ title_attributes }} class="title">{{ label }}</h2>
+  {% endif %}
+  {{ title_suffix }}
+  <div class="description">
+    Today: Browse our editor's hand picked articles!
```
Comment (Y0r) [inline web/themes/custom/newsweek/templates/content/block--views-block--fresh-stories-block-1.html.twig:15]: translatable

File: web/themes/custom/newsweek/templates/content/block--views-block--popular-block-1.html.twig [hunk 8]
```diff
@@ -0,0 +1,25 @@
+
+{%
+  set classes = [
+  'block',
+  'block-' ~ configuration.provider|clean_class,
+  'block-' ~ plugin_id|clean_class,
+]
+%}
```
Comment (Y0r) [inline web/themes/custom/newsweek/templates/content/block--views-block--popular-block-1.html.twig:8]: CS

File: web/themes/custom/newsweek/templates/content/node--newsweek-article--teaser-full.html.twig [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+    'clearfix',
+  ]
+%}
+{%
+  set images_style = {
+  '#theme':      'image_style',
+  '#style_name': 'teaser_full',
+  '#uri':        node.field_main_image.entity.field_media_image.entity.fileuri,
+  }
+%}
```
Comment (Y0r) [inline web/themes/custom/newsweek/templates/content/node--newsweek-article--teaser-full.html.twig:18]: CS

**Review by Y0r — APPROVED** [review]

File: web/themes/custom/newsweek/scss/content/_breaking.scss (hunk 3 above)
Comment (bobik-dev-branck) [reply web/themes/custom/newsweek/scss/content/_breaking.scss:9]: Fixed

File: web/themes/custom/newsweek/scss/content/_breaking.scss (hunk 4 above)
Comment (bobik-dev-branck) [reply web/themes/custom/newsweek/scss/content/_breaking.scss:29]: Fixed

File: web/themes/custom/newsweek/templates/content/block--views-block--breaking-block-1.html.twig (hunk 6 above)
Comment (bobik-dev-branck) [reply web/themes/custom/newsweek/templates/content/block--views-block--breaking-block-1.html.twig:3]: Breaking

File: web/themes/custom/newsweek/scss/content/_breaking.scss (hunk 5 above)
Comment (bobik-dev-branck) [reply web/themes/custom/newsweek/scss/content/_breaking.scss:95]: Fixed

File: web/themes/custom/newsweek/templates/content/block--views-block--fresh-stories-block-1.html.twig (hunk 7 above)
Comment (bobik-dev-branck) [reply web/themes/custom/newsweek/templates/content/block--views-block--fresh-stories-block-1.html.twig:15]: Fixed

File: web/themes/custom/newsweek/templates/content/block--views-block--popular-block-1.html.twig (hunk 8 above)
Comment (bobik-dev-branck) [reply web/themes/custom/newsweek/templates/content/block--views-block--popular-block-1.html.twig:8]: Fixed

File: web/themes/custom/newsweek/templates/content/node--newsweek-article--teaser-full.html.twig (hunk 9 above)
Comment (bobik-dev-branck) [reply web/themes/custom/newsweek/templates/content/node--newsweek-article--teaser-full.html.twig:18]: Fixed

**Pull request merged**